src/Network/Network/AzureFirewall/NetworkRuleCollection/NewAzureFirewallNetworkRuleCollectionCommand.cs

```diff
@@ -50,7 +50,6 @@ public class NewAzureFirewallNetworkRuleCollectionCommand : NetworkBaseCmdlet
         [ValidateSet(
             MNM.AzureFirewallRCActionType.Allow,
             MNM.AzureFirewallRCActionType.Deny,
-            MNM.AzureFirewallRCActionType.Alert,
```

Comment:
This is a breaking change. `Alert` was added in [Network 1.6.0](https://github.com/Azure/azure-powershell/blob/master/src/Network/Network/ChangeLog.md#version-160), and cannot be removed outside of a breaking change release. Please revert.

Reply:
@fanymanea  please review comment from PS team. please discuss you change and revert if it's needed